### ISSUE TITLE
IndexExchange with version 0.6.0

### ISSUE THREAD

**Issue by dapsych0** [prebid/Prebid.js#317]
With the current version (0.7.0) unusable for IndexExchange I have tried to implement version 0.6.0 but it doesn't return a bid cpm.

My setup is as follows:

```
bidder: 'indexExchange',
params: {
    "id": "1",
    "siteID": 158654,
}
```

And the response comes back as follows:

```
bids: Array[1]
  0: Object
    ad: "<iframe src="http://a296.casalemedia.com/ifnotify?c=706A1D&r=6795D023&t=570F898B&u=Vk95WjJzQW9KMmtBQUNFV0N0MEFBQUJ2&m=355e6f146e6979c86410b04374cb5ac4&wp=283&aid=E93DC96094330BF4&tid=D691&s=26BBE&cp=6.43&d=62FF&n=www.site.co.uk&sid=1be7-1cc1-1de3-1de4-23c8-257f-25e3-2604-27e4&epr=658230826" width="0" height="0" frameborder="0" scrolling="no" style="display:none;" marginheight="0" marginwidth="0"></iframe><script language='JavaScript' src='http://tags.mathtag.com/notify/js?exch=cas&id=5aW95q2jLzgvIC9aamhoWWpVMlpUWXRaR0ZqWWkwME5USTBMVGxoWlRndE9ERmlOalJoWVRBNFlqRTMvMzYzODAwNzUwMzI5MDI1NzAzNS8yNDM5NzY2LzEyODE4NDIvMTUvMzBXbi1nXzJZbmYtRVppZ2NRQmZEWUdRUldaUHlyY3pHY0tWeHZROWFkYy8yLzk0NjQ0LzE0NjA1NDc4MzIvMC8xOTMxMzYvMjk4Nzg4NjM5NS8xMzc0NjEvMjU1NTc4LzEvMC8wL1pqaGhZalUyWlRZdFpHRmpZaTAwTlRJMExUbGhaVGd0T0RGaU5qUmhZVEE0WWpFMy8/xWhVFFr6M6_AiJ2QQztOwB-7yzA&sid=1281842&cid=2439766&nodeid=713&price=6.43&bp=g_hhaaaa'></script>"
    adId:"14d31b42dc91a"
    adUnitCode:16
    ad_id:"1"
    adserverTargeting:Object
    bidder:"indexExchange"
    bidderCode:"indexExchange"
    cpm:null
    height:250
    requestTimestamp:1460636043214
    responseTimestamp:1460636043451
    siteID:158654
    size:"300x250"
    statusMessage:"Bid available"
    timeToRespond:237
    width:300
```

Can anyone shed some light on why this is happening?
I have tried switcheroo, which does return a bid, but its very big, something like 200000.00 -- but I guess thats because the sandbox needs the upcoming version of prebid to work correctly?

The index response is:

```
{"seatbid":[{"bid":[{"crid":"7367197","adomain":["​domain.com"],"adid":"7367197","impid":"1","dealid":"​DEALID_STRING","id":"1","ext":{"deal":"25343","dealid":"​DEALID_STRING","dealname":"​DEAL_​NAME","pricelevel":"_510",,"dspid":94,"advbrandid":48063,"advbrand":"Fitbit"},"adm":"<iframe src=\"http://a296.casalemedia.com/ifnotify?c=706A1D&r=9527D00D&t=570F8D70&u=Vk95WjJzQW9KMmtBQUNFV0N0MEFBQUJ2&m=4a8e5fa03842e6eacc2c6ab6e77ed41f&wp=283&aid=E93E460094370BEB&tid=D691&s=26BBE&cp=6.43&d=62FF&n=www.site.co.uk&sid=1be7-1cc1-1de3-1de4-23c8-257f-25e3-2604-27e4&epr=249968445&uds=db\" width=\"0\" height=\"0\" frameborder=\"0\" scrolling=\"no\" style=\"display:none;\" marginheight=\"0\" marginwidth=\"0\"></iframe><script language='JavaScript' src='http://tags.mathtag.com/notify/js?exch=cas&id=5aW95q2jLzgvIC9aamhoWWpVMlpUWXRaR0ZqWWkwME5USTBMVGxoWlRndE9ERmlOalJoWVRBNFlqRTMvMzYzODUyMzcyODI5NDQ2NjE2MC8yNDM5NzY2LzEyODE4NDIvMTUvMzBXbi1nXzJZbmYtRVppZ2NRQmZEU1cxMUJQTlBFM3JFaDhxQmRLcjNuRS8yLzk0NjQ0LzE0NjA1NDc4MzIvMC8xOTMxMzYvMjk4Nzg4NjM5NS8xMzc0NjEvMjU1NTc4LzEvMC8wL1pqaGhZalUyWlRZdFpHRmpZaTAwTlRJMExUbGhaVGd0T0RGaU5qUmhZVEE0WWpFMy8/jWiwpS31Qmm3pehm0z3gcbXUoug&sid=1281842&cid=2439766&nodeid=713&price=6.43&bp=g_hhaaaa'></script><!-- Quantcast Tag -->\n<script type=\"text/javascript\">\nvar _qevents = _qevents || [];\n(function() {\nvar elem = document.createElement(\"script\");\nelem.src = (document.location.protocol == \"https:\" ? \"https://secure\" : \"http://edge\") + \".quantserve.com/quant.js\";\nelem.async = true;\nelem.type = \"text/javascript\";\nvar scpt = document.getElementsByTagName(\"script\")[0];\nscpt.parentNode.insertBefore(elem, scpt);\n})();\nvar label = \"158654.\" + \"http://www.dailymail.co.uk/sport/index.html\".replace(/\\./g,\"_\");\n_qevents.push({\nqacct: \"p-rS1Tb3d-1409Q\",\nlabels: label\n});\n</script>\n<noscript>\n<div style=\"display:none;\">\n<img src=\"//pixel.quantserve.com/pixel/p-rS1Tb3d-1409Q.gif?labels=158654.http://www.site.co.uk/\" border=\"0\" height=\"1\" width=\"1\" alt=\"Quantcast\"/>\n</div>\n</noscript>\n<!-- End Quantcast tag --> "}],"seat":"100933"}],"cur":"USD","id":249968445,"ext":{"x3":[{"id":"39","val":65},{"id":"4","val":69},{"id":"41","val":64},{"id":"47","val":64},{"id":"53","val":1},{"id":"57","val":88},{"id":"65","val":1},{"id":"66","val":66},{"id":"71","val":1},{"id":"81","val":64},{"id":"85","val":78},{"id":"90","val":1},{"id":"94","val":77},{"id":"97","val":67}]}});
```

**Comment by mkendall07** [comment]
@headertag  might be able to help

**Comment by headertag** [comment]
@mkendall07 @dapsych0 this is fixed on master and planned for 0.8.0

0.7.0 is not supported. If you have an immediate need you can use version 0.6.0

**Comment by dapsych0** [comment]
Im trying to use 0.6.0

**Comment by protonate** [comment]
This issue is fixed and released with [0.8.0](https://github.com/prebid/Prebid.js/releases/tag/0.8.0)
